File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_ANDROID/SAMPLE_ANDROID/payapp_android_sample_v3/AppLink/src/main/java/com/udid/applink/model/PACashReceiptCancelResponseData.java
package com.udid.applink.model;

import android.content.Intent;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * 영수증 취소 결과값
 * Create by 김진원
 */
@Getter
@Setter
@ToString(callSuper = true)
public class PACashReceiptCancelResponseData extends PACommonResponseData{


    /**
     * 승인 번호
     */
    String receiptno;


    /**
     * 임의 변수 1, 2
     */
    String var1;
    String var2;

    public PACashReceiptCancelResponseData(Intent data) {
        super(data);
        receiptno = data.getData().getQueryParameter("receiptno");
        var1 = data.getData().getQueryParameter("var1");
        var2 = data.getData().getQueryParameter("var2");
    }




}

File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_IOS/SAMPLE_IOS/payapp_ios_sample_v3/UniversalLink/model/PACashReceiptCancelResponseData.swift
//
//  PACashReceiptCancelResponseData.swift
//  Payapp_iOS_Sample_V3
//
//  Created by 조상현 on 2023/08/28.
//

class PACashReceiptCancelResponseData: PACommonResponseData {
     
    /**
     영수증번호
     */
    var receiptno: String?
    /**
     임의 변수 1, 2
     */
    var var1: String?
    var var2: String?
    
    
    override init(_ data:[String:Any]) {
        super.init(data)
        
        receiptno               = data["receiptno"] as? String
        var1                    = data["var1"] as? String
        var2                    = data["var2"] as? String
    }
    
    func toString() -> String {
        var str = "state:\(state ?? ""), "
        str += "errorMessage:\(errorMessage ?? ""), "
        str += "receiptno:\(receiptno ?? ""), "
        str += "var1:\(var1 ?? ""), "
        str += "var2:\(var2 ?? "")"
        return str
    }
}
